src/web/mod.rs

```diff
@@ -11,7 +11,17 @@ macro_rules! ctry {
         match $result {
             Ok(success) => success,
             Err(error) => {
-                ::log::error!("{}\n{:?}", error, ::backtrace::Backtrace::new());
+                let request: &::iron::Request = $req;
+
+                ::log::error!(
+                    "called ctry!() on an `Err` value in {}:{}:{}: {}\nclient attempted to fetch the route {:?}\n{:?}",
+                    file!(),
+                    line!(),
+                    column!(),
```

Comment:
Doesn't the backtrace print this information already?

Reply:
It probably does, but this makes the information a lot more accessible

Reply:
I don't see why it's necessary? They'd be printed right next to each other, you're duplicating the info.